pkg/cmd/roachtest/tests/ycsb.go

```diff
@@ -32,6 +32,7 @@ func registerYCSB(r registry.Registry) {
 	cpusConfigs := []int{8, 32}
 	cpusWithReadCommitted := 32
 	cpusWithGlobalMVCCRangeTombstone := 32
+	cpusWithUniformDistribution := 32
 
 	// concurrencyConfigs contains near-optimal concurrency levels for each
 	// (workload, cpu count) combination. All of these figures were tuned on GCP
```

Comment:
```go
// All of these figures were tuned on GCP n1-standard instance types.
```

🤔 

(nothing for you to do here, but these numbers might benefit from another look at some point.)

Reply:
The cpu usage looked fairly saturated for most of the runs I looked at, but yeah it wouldn't hurt to fine tune them.